homeassistant/components/aurora_abb_powerone/__init__.py

```diff
@@ -44,6 +68,6 @@ async def async_unload_entry(hass: HomeAssistant, entry: ConfigEntry):
     # It should not be necessary to close the serial port because we close
     # it after every use in sensor.py, i.e. no need to do entry["client"].close()
     if unload_ok:
-        hass.data[DOMAIN].pop(entry.unique_id)
+        hass.data[DOMAIN].pop(entry.entry_id)
```

Comment:
Let's not do this change here so we don't need to change the any platforms in this PR.
```suggestion
        hass.data[DOMAIN].pop(entry.unique_id)
```

Reply:
Ok. Done.